pkg/services/ngalert/database_mig.go

```diff
@@ -104,21 +104,6 @@ func alertInstanceMigration(mg *migrator.Migrator) {
 
 	// create table
 	mg.AddMigration("create alert_instance table", migrator.NewAddTableMigration(alertInstance))
-	mg.AddMigration("add index in alert_instance table on def_org_id, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
+	mg.AddMigration("add index in alert_instance table on def_org_i, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
```

Comment:
Is the change from `def_org_id` to `def_org_i` intentional here? Looks suspicious.